frontend/web/src/components/OrderCard/OrderCard.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styles from './OrderCard.module.scss';
+import { Link } from "react-router-dom";
+
+interface IOrderCardProps {
+    id: number;
+    storeName: string;
+    orderContent: string;
+    distance : number;
+    img?: string;
+    handleOpen: Function;
+}
+
+// TODO: Link to order with accept button and such
+export const OrderCard = ({id, storeName, orderContent, distance, img, handleOpen}: IOrderCardProps) => {
```

Comment:
Skulle kunna ändra "export" till "export default" så att imorten går från "import { OrderCard } from 'components/OrderCard/OrderCard';" till "import { OrderCard } from 'components/OrderCard/OrderCard';"
```suggestion
export default const OrderCard = ({id, storeName, orderContent, distance, img, handleOpen}: IOrderCardProps) => {
```